detox/src/devices/android/ADB.js

```diff
@@ -6,8 +6,8 @@ const EmulatorTelnet = require('./EmulatorTelnet');
 const Environment = require('../../utils/environment');
 
 class ADB {
-  constructor() {
-    this.adbBin = path.join(Environment.getAndroidSDKPath(), 'platform-tools', 'adb');
+  constructor({ adbBin } = {}) {
```

Comment:
where do you pass adbBin to the constructor? What's the usecase for this ?

Reply:
@rotemmiz , was my explanation sufficient? Can we merge this PR?